doc/api/fs.md

```diff
@@ -430,8 +430,11 @@ included in the iteration results.
 added: v10.10.0
 -->
 
-A representation of a directory entry, as returned by reading from an
-[`fs.Dir`][].
+A representation of a directory entry, which can be a file or folder
+within the directory, as returned by reading from an [`fs.Dir`][]. The
```

Comment:
I think mixing _folder_ and _directory_ in the same sentence is confusing. Are they not the same thing?

Reply:
```suggestion
as returned by reading from an [`fs.Dir`][]. The
```

Reply:
Should I put another directory then?